### DISCUSSION TITLE
Error in local setup

### DISCUSSION THREAD

**Discussion by komalverma04** [camicroscope/caMicroscope#828] [Q&A]
Hello all, I was setting the codebase in my local machine.
The steps that i followed are:-
1. Installed docker-desktop
2. Fork and cloned three repos( caMicroscope, distro and Caracal).
3. Change the contex to the fork of yor local Caracal, Mount the cloned caMicroscope directory as a volume.
4. Dockerfile in Caracal repository and delete the line which runs the git clone
5. Ran two commands 1. docker-compose -f caMicroscope.yml build back, 2. docker-compose -f caMicroscope.yml up
after this i started getting error 
![image](https://github.com/camicroscope/caMicroscope/assets/114138604/c82e56b1-e709-44c2-99fd-be6b0817ab90)
![image](https://github.com/camicroscope/caMicroscope/assets/114138604/daec719f-0722-4126-b6fd-7d124c6ddbb5)
Please help!!
Thankyou for your time.

**Comment by gbengaoluwadahunsi** [comment]
Hello @komalverma04 

run this
 docker-compose -f develop.yml build loader --no-cache 

before you run 
5. Ran two commands 1. docker-compose -f caMicroscope.yml build back, 2. docker-compose -f caMicroscope.yml up

I can see that your ca-load is not working, so run the above codes.

**Comment by komalverma04** [comment]
Thank you so much @gbengaoluwadahunsi 
I got that, but my terminal is stuck at the point specified in the image since more than 2 hours.
![image](https://github.com/camicroscope/caMicroscope/assets/114138604/9c500f20-3d40-48f0-ab46-1de9054043ef)
I am just wondering that if it is due to google-drive.json file in credentials directory. So do i have to change this file according to my credentials?

**Comment by gbengaoluwadahunsi** [comment]
ooh, try run the build in pararrel with these codes
DOCKER_BUILDKIT=1 docker-compose -f caMicroscope.yml build

It will make your build time faster.

**Reply by blessdamy** [reply]
Thank you @gbengaoluwadahunsi

**Reply by gbengaoluwadahunsi** [reply]
My pleasure @blessdamy

**Comment by gbengaoluwadahunsi** [comment]
You are welcome
Happy contributions
